integration-tests/gradle/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-# https://gradle.org/release-checksums/
```

Comment:
Can you leave this line? It's pretty handy to know where to cross-check the checksum and where to get the ones during updates.

Reply:
Sorry, missed it. Gradle's `Wrapper` task rewrites properties and I missed this change